operator/operator.go

```diff
@@ -506,12 +506,12 @@ func (o *Operator) getPodToUpdate(pods []v1.Pod, sts *appsv1.StatefulSet, sr Sta
 		return nil, nil
 	}
 
-	prioritizedNodes, err := o.getPrioritizedNodes()
+	prioritizedNodes, cordonedNodes, err := o.getNodes()
```

Comment:
wouldn't it make sense to unify the naming cordoned vs. unschedulable?

Reply:
Yes, changed it to unscheduled midway through the change, forgot to change this variable..